pkgs/development/libraries/tagparser/default.nix

```
@@ -1,29 +1,34 @@
 { stdenv
-, pkgs
+, lib
 , fetchFromGitHub
 , cmake
 , cpp-utilities
 , zlib
+, isocodes
 }:
 
 stdenv.mkDerivation rec {
   pname = "tagparser";
-  version = "9.4.0";
+  version = "11.5.0";
 
   src = fetchFromGitHub {
     owner = "Martchus";
     repo = "tagparser";
     rev = "v${version}";
-    sha256 = "097dq9di19d3mvnlrav3fm78gzjni5babswyv10xnrxfhnf14f6x";
+    hash = "sha256-qgopl32cFQFQTYu9WBOzPeU69J8K49SREX7X0Pw7Als=";
   };
 
   nativeBuildInputs = [ cmake ];
 
-  buildInputs = [
-    cpp-utilities zlib
-  ];
+  buildInputs = [ cpp-utilities zlib ];
```

Speaking of formatting, I'd prefer:

```suggestion
  buildInputs = [
    cpp-utilities
    zlib
  ];
```